index.js

```diff
@@ -1,95 +1,123 @@
 // ***** Question 1 *****
 
+let counter = 1
+counter = 2
+
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 1", "color: red") 
+console.log("%cQuestion 1", "color: red") 
 
-// console.log(counter) 
-// // => 2
-// console.log("%c----------", "color: red") 
+console.log(counter) 
+// => 2
+console.log("%c----------", "color: red") 
 
 
 // ***** Question 2 *****
-
+const name = "Liam"
+// name = "Not Liam"
 // *** Uncomment the lines below to test
 // *** after testing, comment the line causing the error back in 
 // *** otherwise, the error will stop the rest of your code from running
-// console.log("%cQuestion 2", "color: red") 
+console.log("%cQuestion 2", "color: red") 
 
-// console.log(name)
-// // => "Ian"
+console.log(name)
+// => "Ian"
 // name = "Not Ian" 
-// // => TypeError
-// console.log("%c----------", "color: red") 
+// => TypeError
+console.log("%c----------", "color: red") 
 
 
 
 // ***** Question 3 *****
-
+function drinkWater(current_thirst_level) {
+  console.log("Man I Sure am thirsty")
+  current_thirst_level --
+  console.log("Ahh that hits the spot")
+  return current_thirst_level
+}
```

Comment:
Looks good, just a minor nitpick - variable names in Javascript are typically camelCased, so `current_thirst_level` should be `currentThirstLevel`.